app/models/manageiq/providers/infra_manager/metrics_capture.rb

```diff
@@ -78,7 +78,7 @@ def capture_host_targets(ems)
   # @return [Array<Storage>] supported storages
   # hosts preloaded storages and tags
   def capture_storage_targets(hosts)
-    hosts.flat_map(&:storages).uniq.select { |s| Storage.supports?(s.store_type) & s.perf_capture_enabled? }
+    hosts.flat_map(&:storages).uniq.select { |s| s.storage_type_supported? & s.perf_capture_enabled? }
```

Comment:
I was a little confused by this at first, I think because it sounds like we won't create the storage if it isn't supported.

I think this would more accurately be described as `storage_type_supported_for_scan?` or `_for_ssa?`.  Maybe even add the storage type check into `supports?(:smartstate_analysis)` which we already have?